app/src/main/java/com/socketmobile/stockCount/helper/DatabaseHelper.kt

```diff
@@ -71,4 +71,53 @@ fun getStockCountDir(): File {
 
 fun clearStockCountDir() {
     getStockCountDir().deleteOnExit()
+}
+
+fun getFileNameWithExt(c: Context, file: RMFile): String {
+    return if (isConsolidatingCounts(c)) {
+        "${file.fileName}.csv"
+    } else {
+        "${file.fileName}.txt"
+    }
+}
+
+fun getCountsAggregatedContent(c: Context, file: RMFile): String {
+    var content = file.fileContent
+    val lines = content.split("\n")
+    if (lines.isNotEmpty()) {
+        var contentLines = lines.toMutableList()
+        contentLines.removeAt(0)
+        var barcodeList = mutableListOf<String>()
+        var barcodeCountMap = mutableMapOf<String, Int>()
+        contentLines.joinToString(";").split(";").forEach {
+            var components = if (it.contains(",")) {
+                it.split(",")
+            } else {
+                it.split(" ")
+            }
+            if (components.isNotEmpty()) {
+                val barcode = components[0].trim()
+                val count : Int = if (components.size > 1) {
```

Comment:
Please no "magic" constant. Can you replace this by a correctly defined const?

Reply:
- replaced with const